Tasks/InstallSSHKeyV0/task.loc.json

```diff
@@ -128,6 +128,14 @@
       "argumentFormat": ""
     }
   },
+  "restrictions": {
+    "commands": {
+      "mode": "restricted"
+    },
+    "settableVariables": {
+      "allowed": []
```

Comment:
InstallSSHKey task sets some variables - should we add them to allowed?

![image](https://user-images.githubusercontent.com/8779682/109499407-ebd92080-7aa5-11eb-9341-dc39bf887acd.png)